console/metrics.adoc

```diff
@@ -0,0 +1,52 @@
+include::../partials/attributes.adoc[]
+[id="metrics"] 
+= Flow metrics
+
+The following metrics are available to a Prometheus instance when you enable the flow collector using `skupper init --enable-flow-collector`
+
+flows_total:: For tcp protocol this is the total number of connections. 
+For http or http2 protocol this is the total number of distinct requests made.
+
+octets_total:: The total number of bytes processed by the router.
+
+active_flows:: The number of flows currently active including open tcp connections and _in flight_ http requests.
+
+address_last_time_seconds:: The time since a flow was last observed in seconds.
+
+flow_latency_microseconds:: The latency in microseconds, as observed by the first and last router involved in the flow.
```

Comment:
I would also propose that we do not document latency. It is prolifically used in the skupper console and could be useful, but the signal is not intuitive or very well understood. A useful description would be rather technical - I've attempted one out here in some of my v2 work in progress where we carried forward this signal as `legacy_flow_latency_microseconds` (alongside much discouragement from using the signal.) https://github.com/skupperproject/skupper/blob/87d84aaa30ab00d38e398deb23b580af4375bf82/cmd/network-console-collector/README.md#internal-metrics

> The legacy_flow_latency_microseconds metric exposes the Time to First Byte (TTFB) as observed by skupper from the client (listener) and server (connector) sides on TCP connections. This metric shares the same standard set of labels described in the Application Network Traffic section, but adds a direction label to differentiate the listener and connector side TTFB.